client/src/App.js

```diff
@@ -2,14 +2,15 @@ import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Switch} from "react-router-dom";
 import { theme } from "./themes/theme";
-import Navbar from "./components/navbar/navbar";
+import Navbar from "./components/navbar/Navbar";
 import SignAll from "./pages/SignAll";
 import AllProducts from './pages/AllProducts'
 import Shop from "./pages/Shop";
 import "./App.css";
 import { LoginProvider } from "./contexts/LoginContext";
-import Admin from "./pages/admin";
-import {ProtectedRoute} from './components/protectedRoute'
+import Admin from "./pages/Admin";
+import {ProtectedRoute} from './components/protectedRoute';
```

Comment:
space around the {}
and the file should be PascalCase

Reply:
Made the changes.